tests/scene/test_code_edit.h

```diff
@@ -3972,6 +3972,24 @@ TEST_CASE("[SceneTree][CodeEdit] Backspace delete") {
 	code_edit->backspace();
 	CHECK(code_edit->get_text() == "line 1\nline 2\nline 3");
 
+	SUBCASE("[CodeEdit] folding") {
+		code_edit->set_line_folding_enabled(true);
+
+		/* Unfold previous folded line on backspace if caret is at the first column. */
+		code_edit->set_text("");
+		code_edit->insert_text_at_caret("line 1\n\tline 2\n");
```

Comment:
```suggestion
		code_edit->set_text("line 1\n\tline 2\nline 3");
		code_edit->set_caret_line(2);
		code_edit->set_caret_column(0);
```

I think something like this is more clear.